docs/site/MONOREPO.md

```diff
@@ -8,6 +8,7 @@ The [loopback-next](https://github.com/strongloop/loopback-next) repository uses
 | Package                                                                                                             | npm                                   | Description                                                                                                                                                                                                                                                                                      |
 | ------------------------------------------------------------------------------------------------------------------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | [authentication](https://github.com/strongloop/loopback-next/tree/master/packages/authentication)                   | @loopback/authentication              | A component for authentication support                                                                                                                                                                                                                                                           |
+| [booter-lb3app](https://github.com/strongloop/loopback-next/tree/master/packages/booter-lb3app)                     | @loopback/booter-lb3app               | A booter for LoopBack 3 applications to expose t REST API via LoopBack 4.                                                                                                                                                                                                                        |
```

Comment:
typo: `to expose t REST...`-> `to expose REST...`

Reply:
Thanks, I fixed it in a later commit 😄